plugins/DNS/plugin.go

```diff
@@ -0,0 +1,53 @@
+package dns
```

Comment:
I would week the directory name lowercase to be consistent with the others. Can you change it from `plugins/DNS` to `plugins/dns`?